docs-master/Installation.md

```diff
@@ -45,43 +45,45 @@ npm install @nozbe/with-observables
    - Open `ios/YourAppName.xcodeproj` in Xcode
    - Right-click on **Your App Name** in the Project Navigator on the left, and click **New File…**
    - Create a single empty `Swift` file to the project (make sure that **Your App Name** target is selected when adding), and when Xcode asks, press **Create Bridging Header** and **do not remove `Swift`** file then.
-3. **Link WatermelonDB's native library with the Xcode project**:
-
-    **Automatically**
-
-    ```bash
-    react-native link @nozbe/watermelondb
-    ```
-
-    **Or manually**
-
-    If you don't want to use `react-native link`, you can link the library manually:
+3. **Link WatermelonDB's native library with the Xcode project -- manually**:
 
     1. Open your project in Xcode, right click on **Libraries** in the Project Navigator on the left and click **Add Files to "Your Project Name"**. Look under `node_modules/@nozbe/watermelondb/native/ios` and select `WatermelonDB.xcodeproj`
     2. Go to Project settings (top item in the Project navigator on the left), select your app name under **Targets** → **Build Phases** → **Link Binary With Libraries**, and add `libWatermelonDB.a`
 
     For more information about linking libraries manually, [see React Native documentation](https://facebook.github.io/react-native/docs/linking-libraries-ios).
 
-    **Using CocoaPods**
+4. **Link WatermelonDB's native library with the Xcode project -- using CocoaPods**:
```

Comment:
For a few seconds I was convinced that I need to do both steps 3 and 4 here 😉 IMO the previous instructions were a bit more clear about this matter

Reply:
thanks, I'll improve this